tasks/copy_files_to_archive/README.md

```diff
@@ -2,98 +2,10 @@
 
 **Lambda function copy_files_to_archive **
 
-- [Setup](#setup)
-- [Development](#development)
-  * [Unit Testing and Coverage](#unit-testing-and-coverage)
-  * [Linting](#linting)
 - [Deployment](#deployment)
   * [Deployment Validation](#deployment-validation)
 - [pydoc copy_files_to_archive](#pydoc-copy-files)
 
-<a name="setup"></a>
```

Comment:
I really like this change - moving things to a centralized location is a good idea.